MAINTAINERS.yml

```diff
@@ -497,6 +498,7 @@ Documentation:
 Documentation Infrastructure:
   status: maintained
   maintainers:
+    - kartben
```

Comment:
I think infra should be collaboration level first, until relevant changes are made in the area.

Reply:
certainly fine with me

Reply:
Can you update the PR @kartben?